src/eko/matching_conditions/operator_matrix_element.py

```diff
@@ -4,24 +4,67 @@
 |VFNS| evolution.
 """
 
+import functools
 import logging
-import time
 
 import numba as nb
 import numpy as np
 from scipy import integrate
```

Comment:
This was already a bug in the old code - let's fix it now: `src/eko/matching_conditions/operator_matrix_element.py:12:0: W0611: Unused integrate imported from scipy (unused-import)`

General advice: take a look at `poe lint-warnings` - there are some easy fixes, maybe you can tackle the ones related to your new files?